comparision-plugin.js

```diff
@@ -1,6 +1,45 @@
 'use strict';
 
 const semver = require('semver');
+const NPMDependencyChecker = require('ember-cli-version-checker/src/npm-dependency-version-checker');
```

Comment:
Why do we need to deeply require this? Should be `const Checker = require('ember-cli-version-checker')` no?

Reply:
@rwjblue that would work as long as we never call `forEmber`. There's a constraint here that things must use `npm`.

Reply:
Well we control the arguments directly, I'd prefer to use the main export if at all possible. I'm worried about changes internally to that package that _seem_ fine, but actually break this...

Reply:
fixed